recipes/default.rb

```diff
@@ -2,8 +2,16 @@
 # Cookbook Name:: luigi
 # Recipe:: default
 #
+user node['luigi']['user'] do
+  system  true
+  shell   "/bin/false"
+  only_if node['luigi']['setup_user']
+end
 
-include_recipe 'python'
+group node['luigi']['group'] do
```

Comment:
You need to associate the user to this group.
First create the group, then create the user referencing this group.